src/main/window-manager.ts

```diff
@@ -82,7 +82,7 @@ export class WindowManager extends Singleton {
         show: false,
         minWidth: 700,  // accommodate 800 x 600 display minimum
         minHeight: 500, // accommodate 800 x 600 display minimum
-        titleBarStyle: "hidden",
+        titleBarStyle: "hiddenInset",
```

Comment:
This moves OSX window buttons (traffic lights) a bit inset. This align them nicely with topbar history buttons https://www.electronjs.org/docs/api/frameless-window#hiddeninset